src/libraries/System.Net.Security/ref/System.Net.Security.cs

```diff
@@ -127,30 +121,17 @@ public enum ProtectionLevel
         public SslApplicationProtocol(string protocol) { throw null; }
         public System.ReadOnlyMemory<byte> Protocol { get { throw null; } }
         public bool Equals(System.Net.Security.SslApplicationProtocol other) { throw null; }
-        public override bool Equals([System.Diagnostics.CodeAnalysis.NotNullWhen(true)] object? obj) { throw null; }
+        public override bool Equals([System.Diagnostics.CodeAnalysis.NotNullWhenAttribute(true)] object? obj) { throw null; }
         public override int GetHashCode() { throw null; }
         public static bool operator ==(System.Net.Security.SslApplicationProtocol left, System.Net.Security.SslApplicationProtocol right) { throw null; }
         public static bool operator !=(System.Net.Security.SslApplicationProtocol left, System.Net.Security.SslApplicationProtocol right) { throw null; }
         public override string ToString() { throw null; }
     }
-
     public sealed partial class SslCertificateTrust
     {
-        public static SslCertificateTrust CreateForX509Store(
-                                                System.Security.Cryptography.X509Certificates.X509Store store,
-                                                bool sendTrustInHandshake = false) { throw null;  }
-        public static SslCertificateTrust CreateForX509Collection(
-                                                System.Security.Cryptography.X509Certificates.X509Certificate2Collection trustList,
-                                                bool sendTrustInHandshake = false) { throw null; }
-        private SslCertificateTrust() { throw null; }
-   }
-
-    public sealed partial class SslStreamCertificateContext
-    {
-        internal SslStreamCertificateContext() { throw null; }
-        [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
-        public static SslStreamCertificateContext Create(System.Security.Cryptography.X509Certificates.X509Certificate2 target, System.Security.Cryptography.X509Certificates.X509Certificate2Collection? additionalCertificates, bool offline) { throw null; }
-        public static SslStreamCertificateContext Create(System.Security.Cryptography.X509Certificates.X509Certificate2 target, System.Security.Cryptography.X509Certificates.X509Certificate2Collection? additionalCertificates, bool offline = false, SslCertificateTrust? trust = null) { throw null; }
+        internal SslCertificateTrust() { }
```

Comment:
Why internal rather than private as it is in the src?  Is this just what genapi does when we need a non-public ctor in the ref?

Reply:
Yep, that's what genapi does.